src/components/reusable/FeatureCard.tsx

```diff
@@ -0,0 +1,44 @@
+import { Card, Button } from 'antd';
+import { LinkOutlined } from '@ant-design/icons';
+import { FeatureCardProps } from '@/types/reuseableCardProps';
+
+const FeatureCard: React.FC<FeatureCardProps> = ({ icon, title, description, buttonLink }) => {
+  const renderButton = () => {
+    if (buttonLink) {
+      return (
+        <Button
+          className="btn-style"
+          type="primary"
+          href={buttonLink}
```

Comment:
nit ('nit' means that this comment is not blocking review, it's just a suggestion or note): I'd suggest to use `onClick` with `useNavigate` liks you've done for `SiderMenu` cause' `navigate` uses seamess navigation without page reload while `href` attribute for Button triggers page reload on navigation